cmd/build/validation.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/okteto/okteto/pkg/cmd/build"
+	oktetoErrors "github.com/okteto/okteto/pkg/errors"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/okteto/okteto/pkg/registry"
+	"github.com/okteto/okteto/pkg/types"
+	"golang.org/x/sync/errgroup"
+)
+
+// GetServicesToBuild returns the services it has to built because they are not already built
+func (bc *Command) GetServicesToBuild(ctx context.Context, manifest *model.Manifest) ([]string, error) {
```

Comment:
This doesn't sound to me as a function to be within a `validation.go` file. For example, in `buildv2.go` there is a function called `validateOptions` that would fit here.

I think `services.go` would be a better name for this file